src/components/CytoscapeGraph/graphs/GraphStyles.ts

```diff
@@ -9,6 +10,22 @@ export class GraphStyles {
   }
 
   static styles() {
+    const getEdgeColor = (ele: any): string => {
+      const rate = ele.data('rate') ? parseFloat(ele.data('rate')) : 0;
+      if (rate === 0 || ele.data('isUnused')) {
+        return PfColors.Black;
+      }
+      const pErr = ele.data('percentErr') ? parseFloat(ele.data('percentErr')) : 0;
+      // todo: these thresholds should come from somewhere global
```

Comment:
Isn't config() that "somewhere global"? In other words, can this comment go away?

Reply:
flibbedy floo...